.gitignore

```diff
@@ -72,3 +74,27 @@ bin/waagent2.0c
 
 # mac osx specific files
 .DS_Store
+
```

Comment:
wouldn't it be better to move your virtual environment out of the agent's folder?

in case we ever need to add any of those paths to the agent